packages/babel-preset-react-app/dependencies.js

```diff
@@ -95,6 +97,13 @@ module.exports = function(api, opts) {
           // We should turn this on once the lowest version of Node LTS
           // supports ES Modules.
           useESModules: isEnvDevelopment || isEnvProduction,
+          // Undocumented option that lets us encapsulate our runtime, safely
+          // applying it without requiring it be a dependency of the consuming
+          // project
+          // https://github.com/babel/babel/blob/090c364a90fe73d36a30707fc612ce037bdbbb24/packages/babel-plugin-transform-runtime/src/index.js#L35-L42
+          absoluteRuntime: path.dirname(
+            require.resolve('@babel/runtime/package.json')
```

Comment:
We can pass own path here btw I think. See usage in https://github.com/babel/babel/pull/8435.